app/_locales/en/messages.json

```
@@ -2437,6 +2437,9 @@
   "networkNameTestnet": {
     "message": "Testnet"
   },
+  "networkNameZkSync": {
+    "message": "zkSync"
```

This should be `zkSync Era`
```suggestion
    "message": "zkSync Era"
```